src_c/imageext.c

```diff
@@ -414,10 +414,27 @@ image_save_ext(PyObject *self, PyObject *arg)
 }
 
 static PyObject *
-imageext_get_sdl_image_version(PyObject *self, PyObject *_null)
+imageext_get_sdl_image_version(PyObject *self, PyObject *args,
+                               PyObject *kwargs)
 {
-    return Py_BuildValue("iii", SDL_IMAGE_MAJOR_VERSION,
-                         SDL_IMAGE_MINOR_VERSION, SDL_IMAGE_PATCHLEVEL);
+    int compiled = 1;
+
+    static char *keywords[] = {"compiled", NULL};
+
+    if (!PyArg_ParseTupleAndKeywords(args, kwargs, "|p", keywords,
+                                     &compiled)) {
+        return NULL;
+    }
+
+    if (!compiled) {
```

Comment:
I think it'll be neater to switch these around. Not a big issue though. Like this:
```
if (compiled) {}
else {}
```

Reply:
The reason I left it like this for now is that if it's decided to give everything the same default behavior and the same keyword, I only have to change some stuff in 2 or 3 of them, and since `linked` is kinda the opposite of `compiled`, leaving practically the same structure eliminates some intermediate confusion at the moment

Reply:
I believe the commit I just pushed resolves this